devtools-ci-index.yaml

```diff
@@ -898,6 +898,12 @@
             timeout: '30m'
             svc_name: keycloak-server
         - '{ci_project}-{git_repo}':
+            git_organization: almighty
+            git_repo: keycloak
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_run_tests.sh'
+            timeout: '20m'
```

Comment:
Keycloak build without test takes ~30 minutes. So it will be longer with tests. On my laptop the full build with tests takes ~1 hour 20 minutes. It's JavaEE + maven ;) It downloads internet. It should be faster in jenkins but longer than 30 minutes anyway.